internal/services/error_renderer.go

```diff
@@ -41,12 +44,22 @@ func renderError(w http.ResponseWriter, r *http.Request, renderer render.Rendere
 
 func renderNotFoundOrDAOError(w http.ResponseWriter, r *http.Request, err error, resource string) {
 	if errors.Is(err, dao.ErrNoRows) {
-		renderError(w, r, payloads.NewNotFoundError(r.Context(), err))
+		renderError(w, r, payloads.NewNotFoundError(r.Context(), resource, err))
 	} else {
 		renderError(w, r, payloads.NewDAOError(r.Context(), resource, err))
 	}
 }
 
+func renderNewErrorFromClientErr(w http.ResponseWriter, r *http.Request, err error) {
+	if errors.Is(err, clients.ClientErr) {
+		renderError(w, r, payloads.ClientError(r.Context(), err))
+	} else if errors.Is(err, clientshttp.SourcesErr) {
```

Comment:
What about IB error?

Reply:
Yeah let's treat all the three clients the same way:

* generic HTTP
* sources
* image builder